docs/cne-install.md

```diff
@@ -0,0 +1,189 @@
+# SAT Upgrade with CNE Installer
+
+## Upgrade the System Admin Toolkit Product Stream
+
+Describes how to upgrade the System Admin Toolkit (SAT) product
+stream by using the Compute Node Environment (CNE) installer (`cne-install`).
+
+The CNE installer can only upgrade the SAT product stream for this release. See
+[Install the System Admin Toolkit Product Stream](install.md) for installation
+instructions.
+
+Upgrading SAT with `cne-install` is recommended because the process is both
+automated and logged to help you save time. The CNE installer can be used to
+upgrade SAT alone or with other supported products. Refer to the [*HPE Cray EX
+System Software Getting Started Guide (S-8000)*](https://www.hpe.com/support/ex-S-8000)
+for detailed information about `cne-install` and its options.
+
+### Prerequisites
+
+- CSM is installed and verified.
```

Comment:
@JohnNavitskyHPE, these instructions are supposed to cover an install of just SAT with `cne-install` as well as an install with other CSM products. If I understand correctly, I don't think we want to list "the CSM Software Recipe is installed and available" as a prerequisite since that isn't required for an install of just SAT. (If users are installing with other products, they should be referring to the Getting Started Guide and already know about the CSM Software Recipe.)